neuro_extras/main.py

```diff
@@ -118,7 +118,7 @@ def init_aliases() -> None:
     logger.info(f"Added aliases to {toml_path}")
 
 
-NEURO_EXTRAS_IMAGE_TAG = "v20.10.14a1"
+NEURO_EXTRAS_IMAGE_TAG = "v20.9.30a7"
```

Comment:
so the image `v20.10.14a1` is broken, and today's image that we'll build after will work ?

Reply:
that's correct